lib/node_modules/@stdlib/stats/base/dists/triangular/ctor/benchmark/benchmark.js

```diff
@@ -32,16 +33,27 @@ var Triangular = require( './../lib' );
 
 bench( pkg+'::instantiation', function benchmark( bm ) {
 	var dist;
+	var as = new Float64Array(bm.iterations);
+	var bs = new Float64Array(bm.iterations);
+	var cs = new Float64Array(bm.iterations);
 	var a;
 	var b;
 	var c;
 	var i;
+	for ( i = 0; i < bm.iterations; i++ ) {
+		a = uniform(EPS, 10.0);
+		b = uniform(a + EPS, a + 10.0);
+		c = uniform(a, b);
+		as[i] = a;
+		bs[i] = b;
+		cs[i] = c;
+	}
 
 	bm.tic();
 	for ( i = 0; i < bm.iterations; i++ ) {
-		a = ( randu() * 10.0 ) + EPS;
-		b = ( randu() * 10.0 ) + a + EPS;
-		c = ( randu() * ( b-a ) ) + a;
+		a = as[i];
+		b = bs[i];
+		c = cs[i];
 		dist = new Triangular( a, b, c );
```

Comment:
```suggestion
		dist = new Triangular( a[ i%len ], b[ i%len ], c[ i%len ] );
```

After the above changes have been made.